lost/src/main/java/com/mapzen/android/lost/internal/FusedLocationProviderApiImpl.java

```diff
@@ -123,19 +125,25 @@ public boolean isConnected() {
 
   @Override public Location getLastLocation(LostApiClient client) {
     throwIfNotConnected(client);
+    Location location = null;
     try {
-      return service.getLastLocation();
+      location = service.getLastLocation();
     } catch (RemoteException e) {
-      throw new RuntimeException(e);
+      Log.e(TAG, "Error occurred trying to get last Location " + e.getStackTrace());
```

Comment:
There is also Log.e(TAG,message,throwable), wouldn't this be preferrable? Or has that method any downsides I am not aware of?